records_mover/records/hints.py

```diff
@@ -79,26 +82,21 @@ def cant_handle_hint(fail_if_cant_handle_hint: bool, hint_name: str, hints: Reco
 
 
 def csv_hints_from_reader(reader: 'TextFileReader') -> RecordsHints:
-    header = reader._engine.header
-    quotechar = reader._engine.data.dialect.quotechar
-    delimiter = reader._engine.data.dialect.delimiter
-    escape_char = reader._engine.data.dialect.escapechar
+    # https://github.com/pandas-dev/pandas/blob/master/pandas/io/parsers.py#L783
+    # C parser:
+    # https://github.com/pandas-dev/pandas/blob/e9b019b653d37146f9095bb0522525b3a8d9e386/pandas/io/parsers.py#L1903
+    # Python parser:
+    # https://github.com/pandas-dev/pandas/blob/e9b019b653d37146f9095bb0522525b3a8d9e386/pandas/io/parsers.py#L2253
+    # Compression is definitely inferred:
+    # https://github.com/pandas-dev/pandas/blob/e9b019b653d37146f9095bb0522525b3a8d9e386/pandas/io/parsers.py#L425
+    # TODO: But I don't think I'm passing in 'infer' to csv_streamer...
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker